control_plane/src/neon_local.rs

```diff
@@ -1,3 +1,10 @@
+//!
```

Comment:
This file has to go under `src/bin` in order to be treated as a binary automatically by Cargo.
We can change `Cargo.toml` to do that instead, but IMO better to place the file into `src/bin/`

Reply:
Fixed